### ISSUE TITLE
New-AzGalleryImageVersion does not allow creating a SIG Image in cross-tenant scenario

### ISSUE THREAD

**Issue by olayemio** [Azure/azure-powershell#13210]
The New-AzGalleryImageVersion does not allow me to create a SIG image version resource from a VM in another tenant. I am logged into both tenants and my default context is the destination. 

I run the command
```
New-AzGalleryImageVersion -GalleryImageDefinitionName $imgDef -GalleryImageVersionName $imgVer -GalleryName $galleryName -ResourceGroupName $rg -Location "East US" -TargetRegion $targetRegions -Source $imageID
```

The client has permission to perform action 'Microsoft.Compute/galleries/images/versions/read' on scope '$destinationResourceID', however the current tenant $destinationTenant is not authorized to access linked subscription $sourceResourceID.

**Comment by dingmeng-xue** [comment]
Compute team, please help to look into this feature request.

**Comment by olayemio** [comment]
I am the PM for Compute Images. Even though I am logged into both tenants I do not have the permissions for the tenant where my source resides. In CLI, I have access to both tenants. I was asked to submit the issue here.

**Comment by amjads1** [comment]
Per my conversation with @olayemio , looks like there's a separate email thread going on the same issue between Yemi and the powershell team. Assigning this issue to @dingmeng-xue  and removing compute label.

**Comment by dingmeng-xue** [comment]
We are investigating this feature with @olayemio .

**Comment by yashghia** [comment]
Any updates on this issue? We are hitting the same issue while using az cli `az sig image-version create` to copy over images across SIGs in different subscription across tenants.

**Comment by b-mccormack** [comment]
I am also running into this issue when attempting to copy a SIG image version from one tenant to another. It is possible in Azure CLI but not PowerShell.

My understanding of this is limited, but presumably this is because Azure CLI is using the x-ms-authorization-auxiliary header that enables providing tokens for multiple tenants while PowerShell is not?

**Comment by olayemio** [comment]
@b-mccormack Yes that is correct. The PowerShell team is working on that. cc @grizzlytheodore

**Comment by dingmeng-xue** [comment]
Compute team, please check below line
https://github.com/Azure/azure-powershell/blob/a387ffb94e3491de44a4363e82905604f1da3971/src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs#L140
